website/blog/2019-07-02-version-4.1.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Version 4.1.0
+author: Trygve Lie
+authorURL: https://github.com/trygve-lie/
+---
+
+Version 4.1.0 of `@podium/layout` and `@podium/podlet` are now available.
+
+### Assets
+
+This release contain some minor changes to the `.js()` and `.css()` methods in
+both `@podium/layout` and `@podium/podlet` which paves ground for work we are
+doing to improve asset handling and bundling when building microfrontends with
```

Comment:
is it micro-frontends, micro frontends or microfrontends?

Reply:
I guess its microfrontends